bindings/python/test.py

```diff
@@ -75,6 +76,18 @@ def test_torrent_status(self):
 		st2 = self.h.status()
 		self.assertEqual(st2, st)
 
+	def test_status_last_uploaded_dowloaded(self):
+		# we want to check at seconds precision but can't control session
+		# time, wait for next full second to prevent second increment
+		time.sleep(1 - datetime.datetime.now().microsecond / 1000000.0)
+
+		sessionStart = datetime.datetime.now().replace(microsecond=0)
```

Comment:
is this reliable? wouldn't it be more robust to just take a timestamp immediately before and after constructing the session, and perhaps take the average of the two as the session start time.

regardless of how you slice it though, you'll probably need some slack in the test below. I would suggest making sure they are within some number of milliseconds of each other